frontend/apps/reader/modules/readerstyletweak.lua

```diff
@@ -771,6 +771,16 @@ local BOOK_TWEAK_INPUT_HINT = T([[
 
 %2]], _("You can add CSS snippets which will be applied only to this book."), BOOK_TWEAK_SAMPLE_CSS)
 
+local CSS_SUGGESTIONS = {
+    { "-cr-hint: footnote-inpage;", _("When set on a block element containing the target id of a href, this block element will be shown as an in-page footnote.")},
+    { "-cr-hint: non-linear-combining;", _("Can be set on some specific DocFragment (ie. DocFragment[id*=16]) to ignore them in the linear pages flow.")},
+    { "-cr-hint: toc-level1;", _("When set on an element, its text can be used to build the alternative table of contents.")},
+    { "display: run-in !important,", _("When set on a block element, this element content will be inlined with the next block element.")},
+    { "font-size: 1rem !important;", _("1rem will use your default font size")},
```

Comment:
`your` or just `the`?

Reply:
I thought "your" is clearer (the one _you_ have set in koreader), "the" is ambiguous (the book one ? the default for the element ?...)

But, even "default" is ambiguous :/ "the font size you have set in the bottom menu" is long... :\